pkg/config/default.go

```diff
@@ -188,6 +189,25 @@ const (
 	DefaultVolumePluginTimeout = 5
 )
 
+func defaultSigPath() (string, error) {
```

Comment:
If this should exist at all:
```suggestion
// defaultSigPath is an internal implementation detail of defaultConfig. Do not add any other callers.
func defaultSigPath() (string, error) {
```